felles/oidc/src/main/java/no/nav/vedtak/sikkerhet/context/containers/AuthenticationLevelCredential.java

```diff
@@ -4,13 +4,23 @@
 
 public class AuthenticationLevelCredential implements Destroyable {
 
+    private static final Integer AUTHENTICATION_LEVEL_INTERN_BRUKER = 4;
+    private static final Integer AUTHENTICATION_LEVEL_EKSTERN_BRUKER = 4;
+
+    public static final String AUTHENTICATION_LEVEL_ID_PORTEN = "Level" + AUTHENTICATION_LEVEL_EKSTERN_BRUKER;
+
+
     private int authenticationLevel;
     private boolean destroyed;
 
     public AuthenticationLevelCredential(int authenticationLevel) {
         this.authenticationLevel = authenticationLevel;
     }
 
+    public static AuthenticationLevelCredential forInternBruker() {
```

Comment:
Er dette en antagelse? acr claimet er eksplisitt til id-porten og tokenX og finnes ikke i tokene utstedet av azure. Så authentication level begrepet brukes ikke for SBH og systembrukere.

Reply:
Dette er som før hvis du ser på bruken.